docs/content.zh/docs/connectors/table/formats/csv.md

```diff
@@ -29,18 +29,17 @@ under the License.
 {{< label "Format: Serialization Schema" >}}
 {{< label "Format: Deserialization Schema" >}}
 
-[CSV](https://zh.wikipedia.org/wiki/%E9%80%97%E5%8F%B7%E5%88%86%E9%9A%94%E5%80%BC) Format 允许我们基于 CSV schema 进行解析和生成 CSV 数据。 目前 CSV schema 是基于 table schema 推断而来的。
```

Comment:
You have updated the Chinese documentation with English content. The English content can be found in `docs/content/docs/connectors/table/formats/csv.md` - Please make sure that at least the English one has the right documentation

Reply:
Has been updated